Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.py

```diff
@@ -128,7 +130,7 @@ def main() -> None:
         # If user has not set password properties we will get empty string but client require empty list
 
         headers = {
-            "X-RFToken": demisto_params["token"],
+            "X-RFToken": demisto_params["token"]["password"],
```

Comment:
```suggestion
            "X-RFToken": demisto_params.get("token", {}).get("password") or demisto_params.get("token"),
```
Please fix for backward compatibility.